### PR TITLE
Add devices table

### PR DESCRIPTION
**Pull request by justjuju-me**
# Description

- Add devices table to get device id and device token to send notification

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Dangerous changes
- [ ] Chore (change that does not effect how the code works, eg: change color)
- [ ] Tests

### CODE REVIEW

File: app/controllers/api/v1/users/devices_controller.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    module Users
+      class DevicesController < ApplicationController
+        def create
+          user.devices.create!(device_id: params[:device_id], device_token: params[:device_token])
```
Comment (thaisribon) [inline app/controllers/api/v1/users/devices_controller.rb:6]: I think it's better to use device_params with the params.permit function

File: spec/requests/api/v1/users/devices_spec.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+RSpec.describe 'Api::V1::Users::Devices', type: :request do
+  describe 'POST /create' do
+    subject(:request) do
+      post "/api/v1/users/#{user.id}/devices", params: { device_id: 'abcdf', device_token: 'fdcba' }
+    end
+
+    let(:user) { create(:user) }
+
+    it 'returns' do
```
Comment (NickNish09) [inline spec/requests/api/v1/users/devices_spec.rb:11]: I would put a `returns http status ok`, something like that here

File: spec/requests/api/v1/users/devices_spec.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      expect(response).to have_http_status :ok
+    end
+  end
+
+  describe 'When params is missing' do
+    subject(:request) { post "/api/v1/users/#{user.id}/devices" }
+
+    let(:user) { create(:user) }
+
+    it 'returns' do
```
Comment (NickNish09) [inline spec/requests/api/v1/users/devices_spec.rb:22]: same thing

File: app/controllers/api/v1/users/devices_controller.rb (hunk 1 above)
Comment (NickNish09) [reply app/controllers/api/v1/users/devices_controller.rb:6]: rt

**Review by NickNish09 — APPROVED** [review]

**Review by thaisribon — APPROVED** [review]

**Pull request merged**